apps/server/cypress/support/commands.ts

```diff
@@ -32,12 +32,17 @@ Cypress.Commands.add('visitAgg', () => {
   cy.visit('http://localhost:8080/?job_type=agg')
 })
 
-export {}
+Cypress.Commands.add('visitIdentity', () => {
+  cy.visit('http://localhost:8080/?job_type=identify')
```

Comment:
I know I originally implemented this pattern, but we should configure the baseUrl so we don't have to keep specifying it for each visit: https://docs.cypress.io/api/commands/visit